ams/server/src/main/java/com/netease/arctic/server/table/executor/OrphanFilesCleaningExecutor.java

```diff
@@ -260,6 +284,24 @@ private static int clearInternalTableMetadata(UnkeyedTable internalTable, long l
     return 0;
   }
 
+  private static int clearInternalTableIndependentFiles(UnkeyedTable internalTable) {
+    Set<DeleteFile> independentFiles = IcebergTableUtil.getIndependentFiles(internalTable);
+    if (independentFiles.isEmpty()) {
+      return 0;
+    }
+    RewriteFiles rewriteFiles = internalTable.newRewrite();
+    rewriteFiles.set(SnapshotSummary.SNAPSHOT_PRODUCER, CommitMetaProducer.OPTIMIZE.name());
```

Comment:
I've reconsidered and we shouldn't set `SNAPSHOT_PRODUCER` to `OPTIMIZE` here, as we conceptually consider it a `CLEAN` rather than an `OPTIMIZE`. 

Therefore, We can delete this line of code.